src/meta/meta_ckpt.c

```diff
@@ -1021,6 +1021,12 @@ __ckpt_load(WT_SESSION_IMPL *session, WT_CONFIG_ITEM *k, WT_CONFIG_ITEM *v, WT_C
     if (ret != WT_NOTFOUND && a.len != 0)
         ckpt->run_write_gen = (uint64_t)a.val;
 
+    /* This is a recent addition for disaggregated storage. Allow for the value to be missing. */
+    ret = __wt_config_subgets(session, v, "next_page_id", &a);
+    WT_RET_NOTFOUND_OK(ret);
+    if (ret != WT_NOTFOUND && a.len != 0)
+        ckpt->next_page_id = (uint64_t)a.val;
+
```

Comment:
This will assert later when trying to allocate the next page ID, because it would be smaller than `WT_BLOCK_MIN_PAGE_ID`. We can't assert here, because when the follower opens an oligarch table for the first time, it may not yet have the checkpoint info from the primary—and as such, the page ID would not yet be set.